packages/react/src/components/Modal/Modal.js

```diff
@@ -297,7 +306,13 @@ export default class Modal extends Component {
         ref={this.innerModal}
         role="dialog"
         className={`${prefix}--modal-container`}
-        aria-label={modalAriaLabel}
+        aria-label={this.props[`aria-label`] || modalAriaLabel}
+        aria-labelledby={
+          this.props[`aria-labelledby`] ||
+          modalLabel ||
```

Comment:
These should be `id` values versus text values, I believe

Reply:
Yep.
- `aria-label` is text
- `aria-labelledby` is an id (actually, a space-separated list of ids)

Reply:
thanks! should be resolved now